Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -632,6 +632,9 @@ UIView SetupButtons(nfloat width, nfloat height)
 
 		void SetupSelection(UITableView table)
 		{
+			if (table.GestureRecognizers == null)
+				return;
```

Comment:
It looks like when GestureRecognizers is null then there is no point in calling `_tableView.AddGestureRecognizer(new SelectGestureRecognizer());`, otherwise we would always had NRE

Reply:
I'm not sure `table` and `_tableView` are the same reference here, though.

Reply:
You may be right, though. Just discussing. :)

Reply:
It is the same reference, SetupSelection is called only once, easy to verify.

Reply:
We can change _tableView to table and make the method static, nano-optimization =)

Reply:
It would certainly make it more clear. I'm on board with that approach!

Reply:
I have updated PR